tests/config_parse_test.cpp

```diff
@@ -283,3 +285,129 @@ TEST(ConfigParse, ConfigRoot) {
   EXPECT_NO_THROW(flexi_cfg::Parser::parse(
       std::filesystem::path("config_root/test/config_example_base.cfg"), baseDir()));
 }
+
+TEST(ConfigVisitor, JsonConfigVisitor) {
+  setLevel(flexi_cfg::logger::Severity::INFO);
+  auto cfg = flexi_cfg::Parser::parse(std::filesystem::path("config_example16.cfg"), baseDir());
+
+  auto visitor = flexi_cfg::visitor::JsonVisitor();
+  cfg.visit(visitor);
+  std::string json = visitor;
+
+  ASSERT_EQ(
+      "{\"back\":{\"left\":{\"name\":\"back\",\"offset\":[0.15,-9,-0.06,-0.5]},\"right\":{\"name\":"
```

Comment:
nit: Why not use a raw string literal here instead of escaping all of the double quotes?

Reply:
gtest conveniently prints it in escaped form so I copied it over, I can add a raw string but might run a little long as there are no line breaks. (non-pretty)